python/TestHarness/testers/CSVDiff.py

```diff
@@ -21,6 +21,8 @@ def validParams():
         params.addParam('override_rel_err',   [], "A list of customized relative error tolerances.")
         params.addParam('override_abs_zero',   [], "A list of customized absolute zero tolerances.")
         params.addParam('comparison_file', "Use supplied custom comparison config file.")
+        params.addParam('ignore_columns', [], "A list of variables which will not be included in the CSVDiffs operation.")
```

Comment:
```suggestion
        params.addParam('ignore_columns', [], "A list of variables which will not be included in the comparison.")
```
same for the other one